Gems/Atom/RPI/Code/Tests/Material/MaterialSourceDataTests.cpp

```diff
@@ -205,25 +205,25 @@ namespace UnitTest
             "    \"propertyLayout\": {                                           \n"
             "        \"version\": 1,                                             \n"
```

Comment:
does version number require change?

Reply:
No, this isn't used for anything yet. That's what @hershey5045 is working on.